tfx/utils/io_utils.py

```diff
@@ -140,9 +152,19 @@ def load_csv_column_names(csv_file: Text) -> List[Text]:
     return f.readline().strip().split(',')
 
 
-def all_files_pattern(file_pattern: Text) -> Text:
+def files_pattern(files_path: Text,
```

Comment:
I'd actually vote for splitting this up into separate methods, one generic one with a required file pattern param, and an explicit `tfrecord_files_pattern` which calls the generic one. I think it would improve the readability at the callsite, and also be a bit easier to understand (rather than what may seem like an arbitrary default in `*tfrecord*`). What do you think?

Reply:
@andrewsmartin I addressed in [here](https://github.com/yanyuchka/tfx/pull/1/commits/70d59bda8645b093f63c08aa13b0383dae7d8afe)

Thanks again for the great suggestion!

What do you think? Note that I made `files_pattern` to return glob by default. I accept any nits because we want this to be merged :)

Reply:
Looks great, except for one minor nit: https://github.com/yanyuchka/tfx/pull/1/files#r565589397